airbyte-webapp-e2e-tests/cypress/integration/streamTable.spec.ts

```diff
@@ -0,0 +1,131 @@
+import { initialSetupCompleted } from "commands/workspaces";
```

Comment:
This file should go in `integration/connection/` (we can move the other files later)